odata-protocol/meta.yaml

```diff
@@ -6,6 +6,13 @@ previousStage: csd01
 pubdate: 28 February 2024
 pubdateISO: '2024-02-28'
 copyright: © OASIS Open 2024
+archivename: odata-v4.02-csd02
+partname: part1-protocol
```

Comment:
Should we remove `filename` and use these two instead?

Reply:
No, a separate filename leaves more naming flexibility.